aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -575,6 +575,7 @@ SDPBackend select_sdp_backend(sdp_params const& kernel_params) {
     switch (backend) {
       case SDPBackend::cudnn_attention:
         if (sdp::can_use_cudnn_attention(kernel_params, print_debug)) {
+              TORCH_WARN("USING CUDNN SDPA");
```

Comment:
@eqy Was it intentional to add a warning here? This is causing quite a bit of log spew when training models, and doesn't seem to "warn" about anything being wrong.

Reply:
yeah we should remove this, since we are opt in by default now, but I also think this was reverted already on main for internal failures

Reply:
I'm still seeing this on main:
https://github.com/pytorch/pytorch/blob/0344f95c2ea944cc916290097133470f963a5532/aten/src/ATen/native/transformers/cuda/sdp_utils.cpp#L688

But even more importantly I'm seeing it in the 2.4 release branch:
https://github.com/pytorch/pytorch/blob/1cd41997e99ae1722be3fe88e1867af5f6779433/aten/src/ATen/native/transformers/cuda/sdp_utils.cpp#L686

Reply:
https://github.com/pytorch/pytorch/pull/128688